include/seqan3/alphabet/adaptation/char.hpp

```diff
@@ -0,0 +1,254 @@
+template <typename type>
+struct is_char_adaptation :
+    public std::conditional_t<meta::in<char_adaptations, type>::value, std::true_type, std::false_type>
+{};
+
+//!\brief Shortcut for seqan3::detail::is_char_adaptation.
+template <typename type>
+constexpr bool is_char_adaptation_v = is_char_adaptation<type>::value;
+
+//!\}
+} // namespace seqan3::detail
+
+namespace seqan3
+{
+
+//!\addtogroup adaptation
+//!\{
+
+// ------------------------------------------------------------------
+// type metafunctions
+// ------------------------------------------------------------------
+
+//!\brief Type metafunction that shall return the type of an alphabet in char representation. [char specialization]
+//!\tparam char_type One of `char`, `wchar_t`, `char16_t` or `char32_t`.
+//!\sa seqan3::underlying_char
+template <typename char_type>
+    requires detail::is_char_adaptation_v<char_type>
+struct underlying_char<char_type>
+{
+    //!\brief The same type as char_type.
+    using type = char_type;
+};
+
+//!\brief Type metafunction that shall return the type of an alphabet in rank representation. [char specialization]
+//!\tparam char_type One of `char`, `wchar_t`, `char16_t` or `char32_t`.
+//!\sa seqan3::underlying_rank
+template <typename char_type>
+    requires detail::is_char_adaptation_v<char_type>
+struct underlying_rank<char_type>
+{
+    //!\brief An unsigned integer type of the same size as `char_type`.
+    using type = meta::at<detail::char_adaptations_rank_types,
+                          meta::find_index<detail::char_adaptations, char_type>>;
+};
+
+// ------------------------------------------------------------------
+// value metafunctions
+// ------------------------------------------------------------------
+
+//!\brief Value metafunction that shall return the size of an alphabet. [char specialization]
+//!\tparam char_type One of `char`, `wchar_t`, `char16_t` or `char32_t`.
+//!\sa seqan3::alphabet_size
+template <typename char_type>
+    requires detail::is_char_adaptation_v<char_type>
+struct alphabet_size<char_type>
+{
+    //!\brief Smallest unsigned integral type that can hold value;
+    using type = detail::min_viable_uint_t<(std::size_t)std::numeric_limits<char_type>::max() + 1 -
+                                            std::numeric_limits<char_type>::lowest()>;
+    //!\brief The alphabet's size.
+    static constexpr type value =
+        (type)std::numeric_limits<char_type>::max() + 1 - std::numeric_limits<char_type>::lowest();
+};
+
+// ------------------------------------------------------------------
+// free functions
+// ------------------------------------------------------------------
+
+/*!\name Free function wrappers for the char alphabet adaptation
+ * \brief For `char`, `wchar_t`, `char16_t` and `char32_t` do conversion to/from uint types.
+ * \ingroup adaptation
+ * \{
+ */
+/*!\brief Converting char to char is no-op (it will just return the value you pass in).
```

Comment:
emphasise char in the brief section.
Please look at the other documentation spots as well